src/strucclustutils/structcreatedb.cpp

```diff
@@ -113,12 +114,13 @@ void addMissingAtomsInStructure(GemmiWrapper &readStructure, PulchraWrapper &pul
 }
 
 void findInterfaceResidues(GemmiWrapper &readStructure, std::pair<size_t, size_t> res1, std::pair<size_t, size_t> res2,
-                           std::vector<size_t> & resIdx1, std::vector<size_t> & resIdx2)
+                           std::vector<size_t> & resIdx1)
 {
     std::vector<Vec3> coord1, coord2;
     LocalParameters& par = LocalParameters::getLocalInstance();
     for (size_t res1Idx = res1.first; res1Idx < res1.second; res1Idx++) {
-        float shortestDistance = std::numeric_limits<float>::infinity();
+        // float shortestDistance = std::numeric_limits<float>::infinity();
+        float squarethreshold = par.distanceThreshold * par.distanceThreshold;
```

Comment:
Please pull this above the for-loop and mark it const

Reply:
Also it should be squareThreshold

Reply:
yep